_episodes/02-installation.md

```diff
@@ -1,119 +1,192 @@
+- "Install ESMValTool"
+- "Demonstate that the installation was successful"
 keypoints:
-- "ESMValTool is installed from source code that lives in the [GitHub repository](https://github.com/ESMValGroup/ESMValTool)"
-- "All the required packages can be installed using conda and the [environment.yml file](https://github.com/ESMValGroup/ESMValTool/blob/master/environment.yml)"
-- "You can find more information about installation in the [documentation](https://esmvaltool.readthedocs.io/en/latest/getting_started/install.html)"
+- "All the required packages can be installed using conda"
+- "You can find more information about installation in the documentation"
 ---
+## Overview
 
-> ## Attention
-> 
-> * This episode is based on the ESMValTool installation instructions, for more information and advanced cases you can visit the  ESMValTool [documentation](https://esmvaltool.readthedocs.io/en/latest/getting_started/install.html). 
-> * For this episode it is assumed you have knowledge of [git](https://git-scm.com/) you can refresh your knowledge in the corresponding [git carpentries course](http://swcarpentry.github.io/git-novice/). 
-{: .callout}
+In this tutorial we will be using the [Conda](https://conda.io/projects/conda/en/latest/index.html)
+package manager to install the ESMValTool.
+Other installation methods are also available, they can be found in the
+[documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
+ESMValTool also contains diagnostics written in [Julia](https://julialang.org/).
+Because Julia cannot be installed by Conda, we will install Julia separately.
+We will first learn how to install Conda, Julia, and finally the ESMValTool.
+We end this chapter by testing that the installation was successful.
 
-## Obtaining the source code
+## Install Conda
 
-The ESMValTool source code is available on a public GitHub repository:
-https://github.com/ESMValGroup/ESMValTool
+ESMValTool is distributed using [Conda](https://conda.io/). We will be using the Miniconda minimal installer for conda. We suggest a Python 3 based installer, though if you happen to already have Conda installed it should also work with Python 2. For more information about installing conda, see [the conda installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
 
-The easiest way to obtain it is to clone the repository using git. 
-To clone the public repository open a terminal window and type:
+### Linux
 
-~~~
-    git clone https://github.com/ESMValGroup/ESMValTool.git
-~~~
-{: .source}
+1.  Please download Miniconda3 for Linux at [the miniconda page](https://docs.conda.io/en/latest/miniconda.html). The 64 bit version should work on any recent system. If you have problems in the next step(s) you can alternatively try a 32 bit version.
 
-By default, this command will create a folder called ESMValTool containing the
-source code of the tool. 
+2.  Next, run the installer
```

Comment:
Should we add here that you need an open terminal in the same directory where you downloaded miniconda?

Reply:
Added